Gateway/Request/RefundRequest.php

```diff
@@ -46,7 +46,9 @@ public function build(array $buildSubject)
         $payment = $paymentDataObject->getPayment();
         $transactionId = $payment->getAdditionalInformation(self::TRANSACTION_ID) ?:
             $payment->getAdditionalInformation(self::CHARGE_ID);
-        $amountInCents = Util::formatToCents($buildSubject['amount']);
+        $countryCode = $payment->getAdditionalInformation(self::COUNTRY_CODE) ?: 'USA';
+        $creditMemoAmount = $payment->getCreditmemo()->getGrandTotal();
+        $amountInCents = Util::formatToCents($creditMemoAmount);
```

Comment:
if possible introduce currency in formatToCents to make it more future-proof (future when we deal with a currency that is not multiplied by 100 for the lowest unit, not needed right now though)

Reply:
[This commit](https://github.com/Affirm/Magento2_Affirm/pull/81/commits/4b80b2555afb3d2f4a1b1c72c3a4195c65eb729a) introduces currency parameters to the helper functions.